tests_integration/test_installation.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+import subprocess
+from pathlib import Path
+
+from snowcli import __about__
+
+
+# @pytest.mark.integration
+def test_installation(venv_path, test_root_path, snapshot):
+
+    _build_snowcli(venv_path, test_root_path)
+    _install_snowcli_with_external_plugin(venv_path, test_root_path)
+
+    assert _execute_snow_help(venv_path) == snapshot
+    assert _execute_snow_sql(venv_path, test_root_path) == snapshot
+    assert _execute_command_from_external_plugin(venv_path, test_root_path) == snapshot
```

Comment:
Let's split those to separated cases. This will help us get rid of helper and will prevent this test from growing on and on in future.

Reply:
Better not, locally this test takes 90s to execute, build part is really long.

Reply:
Checked execution time on CI, but still 30s

```
33.39s call     tests_integration/test_installation.py::test_installation
```